companies/weekday-com.json

```diff
@@ -0,0 +1,35 @@
+{
+    "slug": "weekday-com",
+    "relevant-countries": [
+        "at",
+        "be",
+        "cz",
+        "dk",
+        "fi",
+        "fr",
+        "de",
+        "hu",
+        "ie",
+        "it",
+        "nl",
+        "pl",
+        "pt",
+        "sk",
+        "sl",
+        "es",
+        "se",
+        "gb"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Weekday.com",
+    "address": "c/o H & M Hennes & Mauritz GBC AB\nMäster Samuelsgatan 46\n106 38 Stockholm\nSweden",
```

Comment:
^^

Reply:
Seriously though! I went through that one list and just kept thinking "Huh, that selection of countries and general website layout seems familiar. Which company could *possibly* be behind this?" :D